cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/ScopeManager.kt

```diff
@@ -638,21 +665,40 @@ class ScopeManager : ScopeProvider {
         call: CallExpression,
         scope: Scope? = currentScope
     ): List<FunctionDeclaration> {
-        val s = extractScope(call, scope)
+        val (s, name) = extractScope(call, scope)
```

Comment:
Can't, since the parameter is already named `scope` and I cannot override it. I could rename the scope parameter to something like startScope or something.